MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/location/LocationAnimatorCoordinatorTest.kt

```diff
@@ -462,7 +461,7 @@ class LocationAnimatorCoordinatorTest {
     locationAnimatorCoordinator.feedNewLocation(Location(""), cameraPosition, true)
     locationAnimatorCoordinator.cancelAllAnimations()
 
-    assertTrue(locationAnimatorCoordinator.animatorArray[ANIMATOR_CAMERA_LATLNG] == null)
+    assertFalse(locationAnimatorCoordinator.animatorArray[ANIMATOR_CAMERA_LATLNG].isRunning)
```

Comment:
```suggestion
    assertFalse(locationAnimatorCoordinator.animatorArray[ANIMATOR_CAMERA_LATLNG].isStarted)
```

Does this test actually verify the behavior? The animator objects in the array are mocked, so they might always default to false, regardless of what happens before. Could you check whether that's the case?

Reply:
Oh, right, let's verify these animators are canceled.

Reply:
Any suggestions on how to verify it? 
Using `verify{locationAnimatorCoordinator.animatorArray[ANIMATOR_LAYER_LATLNG].cancel()}` will cause 
`io.mockk.MockKException: Missing calls inside every { ... } block.` error.

Reply:
This got me thinking, and actually, the animators in the class are not mocks, they are real objects. That's why you cannot verify them. However, they are not started either, because starting happens in a mocked `animatorSetProvider` so just checking `isStarted` flag is not enough because it's always `false`. The test class would need a bigger refactor to remove the dependency on the real animator objects, so I pushed https://github.com/mapbox/mapbox-gl-native-android/pull/437/commits/94d24a1bfe2df309d17d1e3c6652e43e1bf048db as a stopgap. Feel free to remove it and refactor or leave as is.

Reply:
Looking even more at this, I also found a way to this with the current, real object setup. Check out https://github.com/mapbox/mapbox-gl-native-android/pull/437/commits/b58939ffe68eafc0ce2895a775a533d244796d73 and again, feel free to pick one the option that you think works best.

Reply:
It looks good 👍 . Thank you.